paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlSyncTableAction.java

```diff
@@ -118,13 +123,13 @@ public class MySqlSyncTableAction implements Action {
         this.partitionKeys = partitionKeys;
         this.primaryKeys = primaryKeys;
         this.paimonConfig = paimonConfig;
+
+        setDefaultMySqlConfig();
     }
 
-    @Override
-    public void run() throws Exception {
-        StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
-        build(env);
-        env.execute(String.format("MySQL CTAS: %s.%s", database, table));
+    private void setDefaultMySqlConfig() {
+        // https://ververica.github.io/flink-cdc-connectors/master/content/connectors/mysql-cdc.html#connector-options
+        mySqlConfig.putIfAbsent("port", "3306");
```

Comment:
Can we convert `mySqlConfig` to Flink `Configuration`?
Then we can get from it directly.